src/test/java/org/generationcp/middleware/service/impl/study/StudyServiceImplIntegrationTest.java

```diff
@@ -96,4 +98,23 @@ public void testGetStudyDetailsForGeolocation() {
 		Assert.assertEquals(this.study.getName() + " Environment Number 1", studyDetailsDto.getMetadata().getStudyName());
 	}
 
+	@Test
+	public void testGetStudyDetailsForGeolocationWithPI_ID() {
+		final DmsProject environmentDataset =
+			this.testDataInitializer
+				.createDmsProject("Summary Dataset", "Summary Dataset-Description", study, study, DatasetTypeEnum.SUMMARY_DATA);
+
+		final int locationId = 101;
+		final Geolocation geolocation = this.testDataInitializer.createTestGeolocation("1", locationId);
+		this.testDataInitializer
+			.createTestExperiment(environmentDataset, geolocation, TermId.TRIAL_ENVIRONMENT_EXPERIMENT.getId(), "0", null);
+		this.testDataInitializer.addProjectProp(study, TermId.PI_ID.getId(), "", VariableType.STUDY_DETAIL, "1", 6);
+		final StudyDetailsDto studyDetailsDto = this.studyService.getStudyDetailsForGeolocation(geolocation.getLocationId());
+		Assert.assertFalse(CollectionUtils.isEmpty(studyDetailsDto.getContacts()));
```

Comment:
It would be stronger to add assertion on comparison of studyDetails.getContacts().get(0) to the user details retrieved from workbench users table.

Reply:
@darla-leafnode @gelli-leafnode This test is failing in our local environments and also failed sometimes in jenkins. Please check and fix it

Reply:
It is solved by: 
`		this.testDataInitializer.addProjectProp(this.study, TermId.PI_ID.getId(), "", VariableType.STUDY_DETAIL, String.valueOf(user.getPerson().getId()), 6);
`
Not sure how it was merged to master. Even today I run a branch build which I think merge with master first and it run fine. CI is running ok, so I am not so sure how it is working and if we should expend more time verifying if we have a problem running tests in jenkins

Reply:
We will ignore it for now, if you have some time please check and fix it